spec/javascripts/components/story/expanded_story/expanded_story_notes_spec.js

```diff
@@ -62,7 +62,7 @@ describe('<ExpandedStoryNotes />', () => {
   
         const onCreateSpy = sinon.spy();
   
-        const wrapper = shallow(
+        const wrapper = mount(
```

Comment:
What was the reason for this change?

Reply:
Tests were breaking with shallow rendering, so I used mount to access the entire component.